src/modules/subscription/entities/subscription.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from "typeorm";
+import { ISubscription } from "../interfaces/subscription.interface";
+
+@Entity({ name: "subscriptions" })
+export class Subscription implements ISubscription {
+    @PrimaryGeneratedColumn("uuid")
+    id: string;
+
+    @Column({ type: "varchar" })
+    email: string;
+
+    @CreateDateColumn({ name: "created_at", type: "timestamp" })
+    createdAt: Date;
+    @UpdateDateColumn({ name: "updated_at", type: "timestamp" })
```

Comment:
you can read about this https://www.npmjs.com/package/typeorm-naming-strategies#usage, not to do snake_case naming manually